docs/components.md

```diff
@@ -0,0 +1,35 @@
+### Custom Components
+This page will document some of the custom components of transportd and how they can be used by consumers.
+
+##### Validate Headers
+
+The `validate_headers` component can be used to validate the presence of headers and the existence of specific values in those headers.
+It verifies that *any* of the specified headers exist with *any* of the specified values for that header. Each key under `allowed`
```

Comment:
suggestion: I think this is better worded like `verifies that at least one specified header contains at least one specified value`. The current wording suggests to me that any value could be matched under any header.

Reply:
fair point, fixin

Reply:
done